File: app/src/main/java/ru/dmitrybochkov/rates/data/db/AppDatabase.java
package ru.dmitrybochkov.rates.data.db;


import androidx.room.Database;
import androidx.room.RoomDatabase;
import ru.dmitrybochkov.rates.domain.Rate;

@Database(entities = {Rate.class}, version = 1, exportSchema = false)

public abstract class AppDatabase extends RoomDatabase {

    public abstract RateDao rateDao();

}

File: app/src/main/java/ru/dmitrybochkov/rates/data/db/AppDatabase.kt
package ru.dmitrybochkov.rates.data.db


import androidx.room.Database
import androidx.room.RoomDatabase
import ru.dmitrybochkov.rates.domain.Rate

@Database(entities = [Rate::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {

    abstract fun rateDao(): RateDao

}
